docs/modules/operation/pages/deep-dive/provisioning/directed-discovery.adoc

```diff
@@ -2,27 +2,28 @@
 = Directed Discovery
 
 Directed discovery is the process of manually adding nodes to {page-component-title} through the requisition UI.
-Two other methods for manually adding nodes (quick add node and manually adding an interface) are in the process of being deprecated. We do not recommend using these features.
+Two other methods for manually adding nodes (quick adding a node and manually adding an interface) are in the process of being deprecated.
+We do not recommend using these features.
 
-Make sure you complete the tasks in the <<deep-dive/provisioning/getting-started.adoc#provision-getting-started, Getting Started>> section before adding nodes.
+Make sure you complete the tasks in <<quick-start/provisioning/inventory.adoc, Step 3: Provision Inventory>> before adding nodes.
 
 [[requisition-add-nodes]]
-== Add nodes through the Requisition UI
+== Add nodes through the requisition UI
 
-Before adding nodes to a requisition, you must <<deep-dive/provisioning/getting-started.adoc#requisition-create, create a requisition>>.
+Before adding nodes to a requisition, you must <<quick-start/inventory.adoc#requisition-create, create a requisition>>.
 
-. In the web UI, click the gear icon in the top right.
-. In the *Provisioning* area, choose *Manage Provisioning Requisition*.
-. Click the edit icon beside the requisition you want to add nodes to.
+. In the web UI, click the *Gear* symbol in the top-right of the screen.
+. Choose menu:Provisioning[Manage Provisioning Requisition].
+. Click the *Edit* symbol beside the requisition that you want to add nodes to.
 . Click *Add Node*.
-.. {page-component-title} auto-generates the foreign ID used to identify this node.
-. Fill out information in each of the tabs and click *Save*.
-
-* basic information (node label, auto-generated foreign ID, location)
-* path outage (configure network path to limit notifications from nodes behind other nodes, see <<deep-dive/service-assurance/path-outages#ga-service-assurance-path-outage, Path Outages>>)
-* interfaces (add interface IP addresses and services)
-* assets (pre-defined metadata types)
-* categories (label/tag for type of node, e.g., routers, production, switches)
+** {page-component-title} auto-generates the foreign ID used to identify this node.
+. Enter the appropriate information in each of the tabs:
++
+* Basic information (node label, auto-generated foreign ID, location)
+* Path outage (configure network path to limit notifications from nodes behind other nodes, see <<deep-dive/service-assurance/path-outages#ga-service-assurance-path-outage, Path Outages>>)
+* Interfaces (add interface IP addresses and services)
+* Assets (pre-defined metadata types)
+* Categories (label/tag for type of node, e.g., routers, production, switches)
 * meta-data (customized asset information)
```

Comment:
```suggestion
+
[options="header", cols="1,1"]
|===
| Tab
| Settings

| Basic Information
| Required: Auto-generated foreign ID, node label +
Optional: Minion location, building, city

| <<deep-dive/service-assurance/path-outages#ga-service-assurance-path-outage, Path Outage>>
| Parent foreign source, parent foreign ID, parent node label

| Interfaces
| Interface IP addresses, description, SNMP primary, services, actions

| Assets
| Pre-defined metadata types (name, value, actions)

| Categories
| Category name for node type (for example, Routers, Production)

| Meta-Data
| Customized asset information (scope, interface, service, and so on)
|===
```

Reply:
Love the table. I just changed the width of the second column so it wasn't as far right.